public/app/features/alerting/unified/NotificationPolicies.test.tsx

```diff
@@ -636,102 +385,39 @@ describe('NotificationPolicies', () => {
-      alertmanager_config: {
-        receivers: [{ name: 'default' }, { name: 'critical' }],
-        route: {
-          continue: false,
-          receiver: 'default',
-          group_by: ['alertname'],
-          routes: [simpleRoute],
-          group_interval: '4m',
-          group_wait: '1m',
-          repeat_interval: '5h',
-        },
-        templates: [],
-        mute_time_intervals: [muteInterval],
-      },
-      template_files: {},
-    };
-
-    const currentConfig = { current: defaultConfig };
-    mocks.api.updateAlertManagerConfig.mockImplementation((amSourceName, newConfig) => {
-      currentConfig.current = newConfig;
-      return Promise.resolve();
-    });
+    expect(ui.newChildPolicyButton.query()).not.toBeInTheDocument();
+    expect(ui.newSiblingPolicyButton.query()).not.toBeInTheDocument();
+  });
 
-    mocks.api.fetchAlertManagerConfig.mockResolvedValue(defaultConfig);
+  it('Can add a mute timing to a route', async () => {
+    const { user } = renderNotificationPolicies();
 
-    const { user } = renderNotificationPolicies(dataSources.am.name);
-    const rows = await ui.row.findAll();
-    expect(rows).toHaveLength(1);
-    await user.click(ui.editRouteButton.get(rows[0]));
+    await openEditModal(0);
 
     const muteTimingSelect = ui.muteTimingSelect.get();
-    await clickSelectOption(muteTimingSelect, 'default-mute');
-    expect(muteTimingSelect).toHaveTextContent('default-mute');
+    await clickSelectOption(muteTimingSelect, TIME_INTERVAL_NAME_HAPPY_PATH);
+    await clickSelectOption(muteTimingSelect, TIME_INTERVAL_NAME_FILE_PROVISIONED);
 
-    const savePolicyButton = ui.savePolicyButton.get();
-    expect(savePolicyButton).toBeInTheDocument();
+    await user.click(ui.saveButton.get());
 
-    await user.click(savePolicyButton);
+    expect(await screen.findByRole('status')).toHaveTextContent(/updated notification policies/i);
 
-    await waitFor(() => expect(savePolicyButton).not.toBeInTheDocument());
-
-    expect(mocks.api.updateAlertManagerConfig).toHaveBeenCalled();
-    expect(mocks.api.updateAlertManagerConfig).toHaveBeenCalledWith(dataSources.am.name, {
-      ...defaultConfig,
-      alertmanager_config: {
-        ...defaultConfig.alertmanager_config,
-        route: {
-          ...defaultConfig.alertmanager_config.route,
-          mute_time_intervals: [],
-          matchers: [],
-          routes: [
-            {
-              ...simpleRoute,
-              mute_time_intervals: [muteInterval.name],
-              routes: [],
-              continue: false,
-              group_by: [],
-            },
-          ],
-        },
-      },
-    });
+    const policy = (await ui.row.findAll())[0];
+    expect(policy).toHaveTextContent(
+      `Muted when ${TIME_INTERVAL_NAME_HAPPY_PATH}, ${TIME_INTERVAL_NAME_FILE_PROVISIONED}`
+    );
   });
 
   it.skip('Shows an empty config when config returns an error and the AM supports lazy config initialization', async () => {
```

Comment:
I left this skipped for now as well, because I _think_ that when we have a lazy config init case, the default config has no routes, and we don't actually show the UI properly - is anyone able to confirm?

Reply:
AFAIK The idea behind lazy config is as follows:
- If no one edited the config so far, then the AM instance is turned off to save resources, hence GET requests returns the lazy config error
- If there is a POST request with a new config, it means someone wants to use AM and we should turn it on, and apply the config